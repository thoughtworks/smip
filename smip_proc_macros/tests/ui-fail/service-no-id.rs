use smip::*;

#[service]
struct Service;

fn main() {}