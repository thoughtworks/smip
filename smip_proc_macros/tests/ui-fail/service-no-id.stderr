error: Missing field `id`
 --> tests/ui-fail/service-no-id.rs:3:1
  |
3 | #[service]
  | ^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `service` (in Nightly builds, run with -Z macro-backtrace for more info)
